.config/README.md

```diff
@@ -0,0 +1,70 @@
+# Some Config Files Are Here
+
+In here, config files some.
```

Comment:
Wut?

Also, this suggests we could move more here?

Reply:
Yes. In fact, I do in #753.

that sentence was bad on purpose. I'll make it less bad.